packaging/manylinux/python_helper.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+python_nodot="$(echo $PYTHON_VERSION | tr -d '.')"
```

Comment:
@ejguan : Seems TorchData has similar stuff. What are they used to? -- didn't remember seeing simliar thing in domain libraries.

Reply:
Looks like it updates PATH to reflect the right python? I'll test more to see if this is necessary in our case after this PR lands.